spring-cloud-sleuth-zipkin-stream/src/main/java/org/springframework/cloud/sleuth/zipkin/stream/SamplingZipkinSpanIterator.java

```diff
@@ -110,17 +130,17 @@ public void remove() {
 		// A zipkin span without any annotations cannot be queried, add special "lc" to
 		// avoid that.
 		if (span.logs().isEmpty() && span.tags().isEmpty()) {
```

Comment:
it makes little sense to move the comment to that method actually IMO. I'd leave it where it is